ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_search.py

```diff
@@ -295,3 +295,39 @@ def test_paddle_topk(
         sorted=sorted,
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="paddle.where",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_dim_size=1,
+    ),
+)
+def test_paddle_where(
+    *,
+    dtype_and_x,
+    x1,
+    x2,
+    name,
+    on_device,
+    fn_tree,
+    frontend,
+    backend_fw,
+    test_flags,
+):
+    input_dtype, x, x1, x2 = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        backend_to_test=backend_fw,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        condition=x,
+        x1=x1,
+        x2=x2,
```

Comment:
The ivy API refers to them as `x1` and `x2`, but the frontend function you're testing calls them `x` and `y` so that's what they need to be passed in as